tests/conftest.py

```diff
@@ -131,10 +131,10 @@ def parallel(item):
             testname = "%s::%s::%s" % (item.fspath, item.cls.__name__, item.name)
         else:
             testname = "%s::%s" % (item.fspath, item.name)
-        args = ["-n", "1", "python", "-m", "pytest", "--runxfail", "-s",
+        args = ["-n", "1", "python3", "-m", "pytest", "--runxfail", "-s",
                 "-q", testname]
         if nprocs > 1:
-            args.extend([":", "-n", "%d" % (nprocs - 1), "python", "-m", "pytest",
+            args.extend([":", "-n", "%d" % (nprocs - 1), "python3", "-m", "pytest",
```

Comment:
that's risky, as this will break other peopletest.

Reply:
Wee, I agree with that. But if I may ask, what are these cases where this will break?

Reply:
@mloubout would you prefer that now?

Reply:
Well no, that's still hardcoded and bit dangerous. I would use an env var, like `python = os.getenv('PYTHON', 'python')` so that it's trivial to swtich from one to another depending on system setup.